### ISSUE TITLE
[BUG] Got Error when using CaptureAction.SetLiveModeDataContext

### ISSUE THREAD

**Issue by Zrzzzz** [microsoft/axe-windows#1018]
[WpfApp1.zip](https://github.com/microsoft/axe-windows/files/14921938/WpfApp1.zip)
Please check whether the bug has [already been filed](https://github.com/Microsoft/axe-windows/issues).

## Describe the bug

I wrote some code that imitating AccessibilityInsights. To get element info of a Application. But however, I always got "Type 'System.Drawing.Point' cannot be marshalled to a Variant. Type library is not registered". when I using CaptureAction.

Since I've checked the functions be called, but got no idea to solve it. I hope you can answer.

**To Reproduce**

Run code in ZIP.

## Desktop (please complete the following information):

Windows 11.

## Additional context

Priority requested -

**Comment by v-viyada** [comment]
This package is out of scope from our support. Interfaces and classes not described in [AutomationReference.md](https://github.com/microsoft/axe-windows/blob/main/docs/AutomationReference.md) are not supported and may change at any time.
